app/main.py

```diff
@@ -36,10 +36,14 @@
 
 # Load Models
 human_detector = HumanDetector(pretrained=False)
+
+# check for weights location
+location = "app/models/human_detection/weights/human_classification_weights.pkl"
+if not os.path.exists(location):
+    location = "/models/human_detection/weights/human_classification_weights.pkl"
```

Comment:
You never seem to use this var - supposed to go in the line below?

Reply:
Haha oops, will fix when I get home